docs/composition.md

```diff
@@ -0,0 +1,53 @@
+## Composition
+
+`css` can be used in emotion to build styles that can compose with other styles.
+
+```jsx
+import { css } from 'emotion'
+import styled from 'react-emotion'
+
+// Define a class
+const flexCenter = css`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`
+
+// interpolate it where you want to apply the styles
+const flexCenterClass = css`
+  ${flexCenter};
+  flex-direction: column;
+`
+
+// You can also use it in styled and the css prop
+const FlexCenterComponent = styled.div`
+  ${flexCenter};
+`
+
+
+const flexWrap = props => css`
+  flex-wrap: ${props.wrap};
```

Comment:
Suggestion:

Could there be an example with conditional styling based on props? e.g. 

```js
const ButtonCSS = (props) => css`
  display: ${props.block ? 'block' : 'inline-block'};
`
```

I'm sure there are other examples, but this is a start